docs/api/qiskit-ibm-runtime/0.23/options.mdx

```diff
@@ -45,7 +45,7 @@ estimator.options.update(resilience_level=1)
 Refer to [`SamplerOptions`](qiskit_ibm_runtime.options.SamplerOptions "qiskit_ibm_runtime.options.SamplerOptions") and [`EstimatorOptions`](qiskit_ibm_runtime.options.EstimatorOptions "qiskit_ibm_runtime.options.EstimatorOptions") for V2 Sampler and V2 Estimator options, respectively.
 
 <Admonition title="Note" type="note">
-  If an option is not specified, the server default value is used. The default values are subject to change. See the [API Reference](/api/qiskit-ibm-runtime/0.23/options) for the most up-to-date default values.
+  If an option is not specified, the server default value is used. The default values are subject to change.
```

Comment:
This link is meant to go to the current page itself, so I removed the link because it doesn't really make sense.